Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -85,32 +89,38 @@
             <option value="Puerto Rico">Puerto Rico</option>
             <option value="U.S. Virgin Islands">U.S. Virgin Islands</option>
             <option value="Federal Entities">Federal Entities</option>
-            
+
         </select>
 
         <input type="submit" value="Details by State" name="Get Details" class="button1">
     </form>
-    
-    <table border="1" weidth="90%" class="table table-striped">
-        <tr>
-            <th>Jurisdiction</th>
-            <th>Week of allocations</th>
-            <th>1st dose allocations</th>
-            <th>2nd dose allocations</th>
-            @{
-                foreach (QuickType.PfizerVaccineDistributionAllocations pfizerVaccineDistributionAllocations in pfizerVaccineDistributionAllocationss)
-                {
+    @if (pfizerAllocations.Length > 0)
```

Comment:
check length of array to avoid displaying empty table header
Source:-
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions
https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-5.0&tabs=visual-studio

Reply:
Ah, good idea.  Prevent printing out the header if there are no contents.